ci/recipe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set data = load_setup_py_data() %}
+{% set version = data.get('version') or 'placehold' %}
+{% set release = '.'.join(version.split('.')[:2]) %}
+
+package:
+  name: q2-gneiss
+  version: {{ version }}
+
+source:
+  path: ../..
+
+build:
+  script: python setup.py install
+
+requirements:
+  build:
+    - python 3.5*
+    - setuptools
+
+  run:
+    - python 3.5*
+    - setuptools
+    - gneiss >=0.3.0
+    - scikit-bio
+    - numpy
+    - pandas
+    - seaborn
+    - biom-format >=2.1.5,<2.2.0
+    - qiime2 {{ release }}.*
+    - q2-types {{ release }}.*
+    - q2-composition {{ release }}.*
```

Comment:
q2cli isn't a direct dependency of this package (you could use just the Artifact API for example). Once we've got this build going, it will be available however, so end-users really won't notice.